frontend/src/Student/mutations/createPortfolio.graphql

```diff
@@ -0,0 +1,9 @@
+mutation createPortfolio($input: PortfolioInput!) {
```

Comment:
Do we use this one? I might have made it early on. If it's not being used we can delete it.